xds/src/test/java/io/grpc/xds/XdsDependencyManagerTest.java

```diff
@@ -183,9 +167,6 @@ public void tearDown() throws InterruptedException {
       xdsDependencyManager.shutdown();
     }
     xdsClient.shutdown();
-    channel.shutdown();  // channel not owned by XdsClient
```

Comment:
This comment was simply wrong. GrpcXdsTransport shuts down the channel when no longer used.